views/footer.dt

```diff
@@ -1,6 +1,9 @@
 footer
     - import std.datetime : Clock;
     - import std.conv : to;
+    - import pastemyst.db : getPastiesAmount;
+    - import pastemyst.data : Paste;
     - string year = Clock.currTime().year.to!string();
+    - ulong amount = getPastiesAmount!Paste();
     .copyright copyright &copy; #[a(href="https://github.com/codemyst", target="_blank") codemyst] #{year}
-    .paste-amount (amount) currently active pastes 
+    .paste-amount #{amount} currently active pastes 
```

Comment:
you can just do `.paste-amount #{getPastiesAmount!Paste()}` afaik.
also nice solution to this, for some reason I forgot you can call D code in the template files and I wanted to call `getPastiesAmount` before every `render` call :sweat_smile: